server/src/db/schema.ts

```diff
@@ -91,7 +91,7 @@ export const pack = sqliteTable("pack", {
 	itemDocuments: text('item[]_id)').references(()=> item.id),
 	favoritedByDocuments: text('user[]_undefined'),
 	ownerDocuments: text('user[]_undefined'),
-	ownerDocument: text('user_id)').references(()=> user.id),
+	ownerDocument: text('user_id)').references(()=> UserTable.id),
```

Comment:
I'm fine keeping it lowercase 'user' vs doing something like UserTable. Open to either, my example was to show the type / zod stuff that i saw in the T4 Stack example